### PR TITLE
xerox6280: init at 1.0-1

### PR DESCRIPTION
**Pull request by JoeDupuis**
###### Motivation for this change
Add xerox 6280 (DN/DT/N) printers ppd files.
The xerox 6280 Do not print correctly with the generic ones.

###### Things done
It's only ppd files. I only tested it on Nixos, but there is no executable. 
Should work on all linux distribution. 

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions (Should work too, it's only the ppd files, no executable)  
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/cups/drivers/xerox6280/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    mv usr/share $out/
+  '';
+
+
+  meta = with stdenv.lib; {
+    homepage = https://www.xerox.com/;
+    description = "Print drivers for Xerox Phaser 6280 (DN/DT/N)";
+    license = licenses.unfreeRedistributable;
+    platforms = stdenv.lib.platforms.linux;
+    downloadPage = https://www.support.xerox.com/support/phaser-6280/downloads/enus.html?operatingSystem=linux&fileLanguage=en;
```
Comment (samueldr) [inline pkgs/misc/cups/drivers/xerox6280/default.nix]: Unquoted URLs are soon going to be deprecated from *Nixpkgs*. See https://github.com/NixOS/rfcs/pull/45.

The change (and to `meta.homepage`) is simply to quote the URL.

File: pkgs/misc/cups/drivers/xerox6280/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{ stdenv, rpm, cpio} :
+stdenv.mkDerivation {
+  pname = "Xerox-Phaser-6280";
+  version = "1.0-1";
```
Comment (samueldr) [inline pkgs/misc/cups/drivers/xerox6280/default.nix]: From the `meta.downloadPage`:

> Version: 1.0.0.0

```suggestion
  version = "1.0.0.0";
```

Within Nixpkgs we use the upstream version numbering scheme.

**Review by samueldr — CHANGES_REQUESTED** [review]
Hi! :wave: Thanks for your first contribution!

Some changes at first glance:

 * A derivation, lost in directories, won't be useful, it will need an attribute name and reference in `pkgs/top-level/all-packages.nix`.
 * Ideally, if not too much of a burden for you, it would be best that you split the addition of yourself to `maintainers/maintainer-list.nix` into a separate commit.

In addition to the inline comments.

File: pkgs/misc/cups/drivers/xerox6280/default.nix (hunk 2 above)
Comment (twistedjoe) [reply pkgs/misc/cups/drivers/xerox6280/default.nix]: Not sure what happened there. Thought I copy pasted it from upstream. Fixed

Comment (twistedjoe) [reply pkgs/misc/cups/drivers/xerox6280/default.nix]: Not sure what happened there. Thought I copy pasted it from upstream. Fixed

**Pull request closed without merging**